### ISSUE TITLE
New options for the app

### ISSUE THREAD

**Issue by JuanChido** [Wolfteam/GenshinDb#44]
I have some ideas that you could put in your app, such as putting a resin counter, something that sends us a notification about some weapon ascent material or some talent book that we are looking for, when respawn the minerlas, a notification for the reset of interwined fates and fate encounters of Paimon's shop

**Comment by Ayakaitsu** [comment]
The calculator seems to reset each time the app is closed, so would hope that the calculator can store the info, then add notifications when it's the day to farm the stuff:)

**Comment by seyulae** [comment]
Option to customize character in today's ascension materials so I know which character I'm working on :) 

![Screenshot_20210408_190614_com miraisoft genshindb](https://user-images.githubusercontent.com/5961447/114016526-82ff7800-989d-11eb-981a-afd0f58c6eeb.jpg)

**Comment by Wolfteam** [comment]
> d, so would hope that the calculator can store the info, then add notifications when it's the day to farm the stuff:)

Are you using the latest version of the app ? (1.1.0)

**Comment by Ayakaitsu** [comment]
> > d, so would hope that the calculator can store the info, then add notifications when it's the day to farm the stuff:)
> 
> Are you using the latest version of the app ? (1.1.0)

Oh, am using 1.0.10... I see it's latest version on google playstore though

**Comment by Wolfteam** [comment]
> > > d, so would hope that the calculator can store the info, then add notifications when it's the day to farm the stuff:)
> > 
> > 
> > Are you using the latest version of the app ? (1.1.0)
> 
> Oh, am using 1.0.10... I see it's latest version on google playstore though

Yeah, the 1.1.0 is a stagged rollout, it will be available for everyone by the end of this week

**Comment by Ayakaitsu** [comment]
> > > > d, so would hope that the calculator can store the info, then add notifications when it's the day to farm the stuff:)
> > > 
> > > 
> > > Are you using the latest version of the app ? (1.1.0)
> > 
> > 
> > Oh, am using 1.0.10... I see it's latest version on google playstore though
> 
> Yeah, the 1.1.0 is a stagged rollout, it will be available for everyone by the end of this week

Thats great! Thank you:)
